canonical/jaas.ai
# feat: Remove access to dashboard from jaas.ai

## Done

- Removed all access points to juju-dashboard from jaas.ai
- Removed all remaining artifacts related to juju-dashboard

## QA

- Pull code
- Run `dotrun`
- Open http://0.0.0.0:8029
- Make sure that "/models", "/models/<path:path>", "/controllers", "/controllers/<path:path>", "/config.js", "/manifest.json", "/version.json", "/ghost-bundle.svg" routes are not accessible

## Details

- https://warthogs.atlassian.net/browse/WD-7481
